packages/icons/src/index.ts

```diff
@@ -221,7 +226,7 @@ const SystemIcons = {
   World,
 };
 
-const Pictograms = { Add, Cluster, Email, Globe, Location, Monitoring };
+const Pictograms = { Add, Email, Globe, Location, Monitoring };
```

Comment:
What's the difference between a pictogram, an illustration, and a system icon? Doc strings that explain the categories would clear up any confusion and ensure consistent use.

Reply:
That's a good suggestion. When these sets were documented in Figma it was clearer, but as these have been reorganized it's becoming less clear. From the Design System v1 (descriptions are mine),

System Icons are small vector images containing a single color intended for use at small sizes (8px -32px).

<img width="692" alt="Screenshot 2023-05-24 at 2 54 50 PM" src="https://github.com/cockroachdb/ui/assets/397448/d8854eea-2f13-4481-8f69-3ec4b39f3e96">

Pictograms (or pictogram icons) are vector images containing two colors (a solid fill color and a transparent fill color) for use at medium sizes (36px - 56px).

<img width="592" alt="Screenshot 2023-05-24 at 2 59 39 PM" src="https://github.com/cockroachdb/ui/assets/397448/3cdd4666-4972-4c28-9f92-dbdee4b640cc">

Illustrations are larger more complex vector images containing multiple (3 or more) fill colors for use at a large size (the size it was designed for)

<img width="849" alt="Screenshot 2023-05-24 at 3 01 52 PM" src="https://github.com/cockroachdb/ui/assets/397448/8e2a4da6-2a6a-4ebb-9e0e-ec4b0ff9d93a">

The figma project [CRDB-Design_System-CURRENT](https://www.figma.com/file/lBTLTVQlvBDi5JOiCPnx2i/CRDB-Design_System-CURRENT?type=design&node-id=7874%3A0&t=mVLS6aUuH4UGU6b4-1) doesn't yet contain all of the images that we are using in the app and so I'm not sure about the future of these categories. That said I can add comments and references to the v1 of the design system where these grouping (and associated components) came from.